exercises/19_sum_time_many.rb

```diff
@@ -0,0 +1,38 @@
+def sum_time(time_list)
+  time_arr_1 = time_list[0].split(":").map {|el| el.to_i}
+  time_sum = Time.new(2021, 5, 1, time_arr_1[0], time_arr_1[1], time_arr_1[2])
```

Comment:
Don't hardcode month, year, day instead think of any other logic

Reply:
should I use Time.parse to avoid hard coding?

Reply:
Yes